packages/lit-dev-content/site/docs/v2/ssr/client-usage.md

```diff
@@ -117,6 +117,6 @@ The HTML snippet below includes an optional strategy to hide the body until the
 ```
 
 ### Combined example
-This example shows a strategy that combines both the `lit/experimental-hydrate-support.js` and the `template-shadowroot` polyfill loading and serves a page with a SSRed component to hydrate client-side.
+This example shows a strategy that combines both the `@lit-labs/ssr-client/lit-element-hydrate-support.js` and the `template-shadowroot` polyfill loading and serves a page with a SSRed component to hydrate client-side.
 
-[Lit SSR in a Koa server](https://stackblitz.com/edit/lit-ssr-global?file=src/server.js)
+[Lit SSR in a Koa server](https://stackblitz.com/edit/lit-ssr-global-ptowx2?file=src/server.js)
```

Comment:
I updated the currently linked stackblitz example too so no need to update this.

Reply:
Reverted! Thank you so much!